### DISCUSSION TITLE
Clang version 16 support ?

### DISCUSSION THREAD

**Discussion by v4Corg** [android/ndk#1869] [Q&A] [answered]
Hi there, not sure if my question is placed in the right place.

The current(?) version of ndk (25c) comes with clang 14.0.7 as far as i can tell. 
Is there a way to upgrade clang.exe that to version 16 (at least for our sources to compile) ?

Thank you,
Roman

**Comment by jfgoog** [comment, marked as answer]
Unfortunately, there is no easy way for you to upgrade the NDK's version of clang.

It looks like NDK r26 will probably have clang 17 or newer, so our recommendation is to wait for that.

**Reply by v4Corg** [reply]
thank you
